File: src/1.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
int main(){
    char *line = NULL;
    size_t len = 0;
    ssize_t read;
    int max = 0;
    int cur = 0;
    while((read = getline(&line, &len, stdin)) != -1){
        if(!strcmp(line, "\n")){
            max = cur > max ? cur : max;
            cur = 0;
            continue;
        }
        cur += atoi(line);
    }
    printf("%d\n", max);
    return 0;
}

File: src/1.rs
use std::io;
fn main(){
    let lines = io::stdin().lines();
    let mut max = 0;
    let mut cur = 0;
    for line in lines {
        if line.as_ref().unwrap().is_empty(){
            max = if cur > max { cur } else { max };
            cur = 0;
            continue;
        }
        cur += line.unwrap().parse::<i32>().unwrap();
    }
    println!("{}", max);
}
